ctrd/container.go

```diff
@@ -55,7 +56,7 @@ func (c *Client) ContainerStats(ctx context.Context, id string) (*containerdtype
 
 // containerStats returns stats of the container.
 func (c *Client) containerStats(ctx context.Context, id string) (*containerdtypes.Metric, error) {
-	if !c.lock.Trylock(id) {
+	if !c.lock.TrylockWithTimeout(id, defaultTrylockTimeout) {
 		return nil, errtypes.ErrLockfailed
```

Comment:
This part would return an error of 
```
failed: rpc error: code = Unknown desc = failed to stop container "c69994fce9611e75ca826c644edb8f4dbc32a2592afb66dd34e1d4a2d9a76c0e": failed to destroy container c69994fce9611e75ca826c644edb8f4dbc32a2592afb66dd34e1d4a2d9a76c0e: lock failed"
```

While I think some users of PouchContainer feel a little bit unclear about the error message, could we add code like 
```
// LockFailedError is constructing a much more readable lock failed error.
func LockFailedError(containerID string) error {
	return errors.Wrapf(errtypes.ErrLockfailed, "container %s is accessed by other request and please try again", containerID)
}
```

Reply:
I think the error will be disappeared after this PR 😢